contracts/factory/K1ValidatorFactory.sol

```diff
@@ -60,7 +60,7 @@ contract K1ValidatorFactory is Stakeable {
         IERC7484 registry
     ) Stakeable(factoryOwner) {
         require(
-            !(implementation == address(0) || k1Validator == address(0) || address(bootstrapper) == address(0) || address(registry) == address(0)),
+            !(implementation == address(0) || k1Validator == address(0) || address(bootstrapper) == address(0) || address(registry) == address(0) || factoryOwner == address(0)),
```

Comment:
registry could be zero. if we have a setter to set it later?

Reply:
I didn't touch that part, I can remove the check if necessary

Reply:
we don't have setters. registry is immutable

Reply:
okay

Reply:
maybe if factory is ownable we should be able to change the registry.